controller/SynchronisationApi.php

```diff
@@ -67,10 +71,18 @@ public function fetchEntityChecksums()
 
             $type = $this->getRequestParameter(self::PARAM_TYPE);
             $params = $this->hasRequestParameter(self::PARAM_PARAMETERS) ? $this->getRequestParameter(self::PARAM_PARAMETERS) : [];
+            $this->getSyncParamsValidator()->validate($params);
```

Comment:
Shouldn't `SynchronisationApi::fetchClassDetails` and `SynchronisationApi::getDeliveryTest` also check for compatible VM version?